knowledge-portal/source/initiate-push-to-central.sh

```diff
@@ -7,6 +7,12 @@ git config --global user.email "PlatformTeam@detecttechnologies.com"
 git config --global user.name "Detect Gitlab Bot"
 git clone $CENTRAL_GIT_PUSH_URL /root/central
 
+# Install git lfs
+cd /root/central
+apt-get update -qq && apt-get install -y -qq git-lfs
+git lfs install
+git lfs track "*.jpg" "*.jpeg" "*.png" "*.gif" "*.svg" "*.ico" "*.mp3" "*.mp4" 
```

Comment:
Is it really needed to run git lfs install and track on every commit? Tracking would persist in the .gitattribute file.

Reply:
You're right, removed those lines.